src/components/JobWizard/JobWizardSteps/JobWizardParameters/JobWizardParameters.js

```diff
@@ -52,13 +52,15 @@ const JobWizardParameters = ({ formState, stepIsActive }) => {
       </div>
       {hyperParametersAreEnabled && (
         <div className="form-row">
-          <FormRadio name={parametersFromPath} value={PARAMETERS_FROM_UI_VALUE} label="From UI" />
+          <FormRadio id={`${parametersFromPath}-${PARAMETERS_FROM_UI_VALUE}`} name={parametersFromPath} value={PARAMETERS_FROM_UI_VALUE} label="From UI" />
```

Comment:
Can be applied the same logic as in the FormInput (using name prop inside the component as an id)

Reply:
It is the same logic. I don't use the "name" prop for the ID since I don't want the ID with "." char.

Reply:
What is wrong with having "." char?

Reply:
trying to make the same naming using "-" as separators. I am OK with keeping id as prop and not use name